apps/application-system/api/src/app/modules/application/files/utils/pdf.ts

```diff
@@ -295,7 +292,38 @@ export async function generateResidenceChangePdf(
     PdfConstants.NORMAL_FONT,
     PdfConstants.SMALL_FONT_SIZE,
     PdfConstants.SMALL_LINE_GAP,
-    `Undirritaður/uð, ${applicant.fullName}, hefur heimilað fyrirspurn í Þjóðskrá og staðfest með undirritun sinni að ofangreindar upplýsingar séu réttar.`,
+    `Undirritaður málshefjandi, ${applicant.fullName}, heimilaði fyrirspurn og uppflettingu í forsjárgögnum hjá Þjóðskrá Íslands þann ${nationalRegistryLookupDate} kl. ${nationalRegistryLookupTime}.`,
+  )
+
+  doc.moveDown()
+
+  addToDoc(
+    PdfConstants.NORMAL_FONT,
+    PdfConstants.SMALL_FONT_SIZE,
+    PdfConstants.SMALL_LINE_GAP,
+    `Niðurstaða uppflettingar var að ${currentParent.parentName} (${currentParent.nationalId}) og ${futureParent.parentName} (${futureParent.nationalId}) eru með sameiginlega forsjá með eftirfarandi börnum:`,
+  )
+
+  doc.moveDown()
+
+  childrenAppliedFor.map((c, i) =>
+    addToDoc(
+      PdfConstants.NORMAL_FONT,
+      PdfConstants.SMALL_FONT_SIZE,
+      i === childrenAppliedFor.length - 1
+        ? PdfConstants.LARGE_LINE_GAP
+        : PdfConstants.NO_LINE_GAP,
+      `- ${c.fullName} (${formatSsn(c.nationalId)})`,
+    ),
+  )
```

Comment:
Not sure if we should render all children here or just the selected ones that are in the contract.

Reply:
Removed